src/proto.rs

```diff
@@ -576,22 +639,62 @@ impl RequestRegistrationExtensionsBuilder {
     ///
     /// assert_eq!(extensions.cred_protect, Some(cred_protect));
     /// ```
-    pub fn cred_protect(mut self, cred_protect: RequestCredProtectExtension) -> Self {
+    pub fn cred_protect(mut self, cred_protect: CredProtect) -> Self {
         self.0.cred_protect = Some(cred_protect);
         self
     }
+
+    /// Set the credential blob extension options
+    ///
+    /// # Example
+    /// ```
+    /// # use webauthn_rs::proto::{RequestRegistrationExtensions, CredBlobSet};
+    /// let cred_blob = vec![0xde, 0xad, 0xbe, 0xef];
+    /// let extensions = RequestRegistrationExtensions::builder()
+    ///     .cred_blob(cred_blob.clone())
+    ///     .build();
+    ///
+    /// assert_eq!(extensions.cred_blob, Some(CredBlobSet::from(cred_blob)));
+    /// ```
+    pub fn cred_blob(mut self, cred_blob: Vec<u8>) -> Self {
```

Comment:
Should this be "set_cred_blob"?

Reply:
This is a good question. My modeling of the builder was after how `tide` does there's, in which the names of the setters are the names of the fields they set. I suppose there's a bit of ambiguity if we do `set_cred_blob`, in whether "`set`" refers to `set`ting the field in the struct or `set`ting the cred blob data for the authenticator

Reply:
If the latter, am I correct in assuming that we would also want `cred_blob` -> `get_cred_blob` for the other builder? One could also argue for `set_cred_protect`, I suppose.

Reply:
Ahh yeah. Well, we also have just "cred_protect" here too. Honestly it's much-over-muchness, so I think this is fine :) I retract my comment :)